### ISSUE TITLE
dataset composition

### ISSUE THREAD

**Issue by 704051419** [Jia-Research-Lab/PFENet#13]
Sorry to bother again, the train_list you provide consist of 'voc_train' and 'sbd_train'. I wonder if 'sbd_val' should be included too(like 'trainaug' in Deeplab).  I did not find the detailed description in OSLSM.
And i am still a little confused about the image number. 'sbd_train+'voc_train'=8284, why the number is above 5900 in your setting?
Not sure if i miss some preprocess.

**Comment by tianzhuotao** [comment]
We follow the instructions in http://home.bharathh.info/pubs/codes/SBD/download.html to use sbd_train that excludes all images in the VOC validation set. The sbd_val set overlaps with the validation set of VOC.

The final list (5900) is derived from the SBD train (5623) + VOC train (1464) by removing all duplicate images.

**Comment by 704051419** [comment]
Thanks for your reply, i just check the setting of CANet, after removing the small object and duplicate images, the training set contains 10092 images, which is far more than your settings. 
 I think it is the subset of 'trainaug.txt' in deeplab which contains 10582 no duplicated  images( voc_train(1464)+sbd_train(6820)+sbd_val(2298)). Intuitively, a bigger train_set may boost the performance. I will try to train your model in that way and check the performance.Thanks!

**Comment by tianzhuotao** [comment]
Thank you.

Strictly speaking, because few-shot segmentation uses the validation set of VOC for evaluation, images contained in the validation set should be excluded. It is also what SBD webpage emphasizes. But, I guess the performance gap will not be very large because few-shot segmentation evaluates only the novel classes. If so, training with 10582 images (trainaug) is also okay.
